packages/widgets/src/lib/components/InferenceWidget/shared/WidgetOutputConvo/WidgetOutputConvo.svelte

```diff
@@ -6,7 +6,7 @@
 
 	export let modelId: string;
 	export let messages: Array<{
```

Comment:
could we export `type Message` & use it here?

Reply:
something like this? https://github.com/huggingface/huggingface.js/pull/493/commits/10dbbc579607cd9b827f3f754f502ffdd561332c